src/components/forms/schemas/TenYearAnniversary.ts

```diff
@@ -0,0 +1,81 @@
+import * as z from 'zod';
+
+import { stringFieldSchema } from './utils';
+import { containURL } from '../../../utils';
+
+const MAX_TEXT_LENGTH = 255;
+const MAX_TEXT_AREA_LENGTH = 2000;
+
+export type TenYearAnniversaryData = z.infer<typeof TenYearAnniversarySchema>;
+
+export const TenYearAnniversarySchema = z.object({
+  // Contact Information
+  firstName: stringFieldSchema('First Name', { min: 1, max: 40 }).refine(
+    value => !containURL(value),
+    'First name cannot contain a URL'
+  ),
+  lastName: stringFieldSchema('Last Name', { min: 1, max: 80 }).refine(
+    value => !containURL(value),
+    'Last name cannot contain a URL'
+  ),
+  email: z.string().email({ message: 'Invalid email address' }),
+  company: stringFieldSchema('Name of organization or entity', {
+    max: MAX_TEXT_LENGTH
+  })
```

Comment:
This should have a min 1 since it is a required field
```suggestion
  company: stringFieldSchema('Name of organization or entity', {
    min: 1,
    max: MAX_TEXT_LENGTH
  })
```